app/pages/utils/config.yaml

```diff
@@ -284,14 +289,18 @@ carrier:
   retrofitting: Retrofitting
 
   # remaining
+  Battery Storage: Battery Storage
+  H2 Fuel Cell: H2 Fuel Cell
+
```

Comment:
Have added these technologies as a quick fix to get the app work. That is a temporary solution, and the config definitely will need a throughout clean-up in future. But for not that should be allright